common/types/admin.go

```diff
@@ -26,6 +26,13 @@ type AddSearchAttributeRequest struct {
 	SecurityToken   string                      `json:"securityToken,omitempty"`
 }
 
+func (v *AddSearchAttributeRequest) SerializeForLogging() (string, error) {
+	if v == nil {
+		return "", nil
+	}
```

Comment:
tbh I might just put this in SerializeRequest since it really should be the same every time.

Reply:
Typed nil. Keep as-is.

Reply:
yea.  typed nils are worth avoiding on principle I think